### PR TITLE
updated what's new for 2.7 with more k3s GA

### LINKED ISSUE #7376: Update k3s section in What's New
From @leflambeur:

### k3s provisioning is GA (x86 Only)

- Provisioning K3s clusters on x86 clusters has graduated to GA! The fully compliant Kubernetes distribution is simplified, secure and at less than 60mb is perfect for Edge.

Based on branching, need to update both `master` and `release-2.7.0`

### PR DESCRIPTION
**Pull request by gaktive**
Signed-off-by: Gary A Korhonen <gkorhonen@suse.com>

### Summary
Fixes #7376 

### Occurred changes and/or fixed issues

Updated What's New for 2.7.0

### Screenshot/Video

Note to self: this needs to go in both `master` and `release-2.7.0`

### CODE REVIEW

File: shell/content/docs/en-us/whats-new.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ installed by administrators. Extensions are provided as Helm charts. A new top-l
 
 - Authentication for OCI-based registries is now supported. Note that the structure of the fleet.yaml is the same and the credentials are provided as a Kubernetes secret, which is described in the Private Helm Repo box in the Repo Structure docs.
 
-### k3s provisioning is GA
+### k3s provisioning is GA (x86 Only)
```
Comment (deniseschannon) [inline shell/content/docs/en-us/whats-new.md:16]: Can we move this to the top?

File: shell/content/docs/en-us/whats-new.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ installed by administrators. Extensions are provided as Helm charts. A new top-l
...
 
-- You can now provision clusters that use k3s. 
+- Provisioning K3s clusters on x86 clusters has graduated to GA! The fully compliant Kubernetes distribution is simplified, secure and at less than 60mb is perfect for Edge.
 
 ---
 title: What's New in 2.6
```
Comment (deniseschannon) [inline shell/content/docs/en-us/whats-new.md:21]: When are we branching for 2.7 and 2.6 in UI? I only want "what's new in  2.7" in 2.7 and "what's new in 2.6" in 2.6, I wouldn't want it in both.

File: shell/content/docs/en-us/whats-new.md (hunk 1 above)
Comment (gaktive) [reply shell/content/docs/en-us/whats-new.md:16]: Sure thing.

File: shell/content/docs/en-us/whats-new.md (hunk 2 above)
Comment (gaktive) [reply shell/content/docs/en-us/whats-new.md:21]: I'll remove the 2.6.x bits from here for 2.7, though it may have to linger in 2.6.x (somehow).

Comment (gaktive) [reply shell/content/docs/en-us/whats-new.md]: Need to cherry pick this PR once done into `release-2.7.0`. Need to think about what to do for 2.6.x releases.

**Review by deniseschannon — APPROVED** [review]

File: shell/content/docs/en-us/whats-new.md (hunk 2 above)
Comment (richard-cox) [reply shell/content/docs/en-us/whats-new.md]: FWIW 

`master` --> go forward branch (post 2.7.0)
`release-2.7.0` --> 2.7.0 branch
`release-2.6.9` --> 2.6.9 branch (which will be the base for anything post 2.6.9)

The `release-2.6.9` branch won't contain any details of 2.7.0

**Pull request merged**